golang/cosmos/x/vbank/types/params.go

```diff
@@ -130,3 +156,18 @@ func validatePerEpochRewardFraction(i interface{}) error {
 
 	return nil
 }
+
+func validateAllowedMonitoringAccounts(i interface{}) error {
+	v, ok := i.([]string)
```

Comment:
@michaelfig I am a bit confused about why the interface and then interface reflection are necessary. Do you mind sharing it with me?